generators/entity-client/index.js

```diff
@@ -1,60 +1,19 @@
 /* eslint-disable consistent-return */
-const chalk = require('chalk');
 const EntityClientGenerator = require('generator-jhipster/generators/entity-client');
-
+const writeFiles = require('./files').writeFiles;
 
 module.exports = class extends EntityClientGenerator {
     constructor(args, opts) {
         super(args, Object.assign({ fromBlueprint: true }, opts)); // fromBlueprint variable is important
-
-        const jhContext = this.jhipsterContext = this.options.jhipsterContext;
```

Comment:
these are required actually

Reply:
When using this snippet with entity-client, blueprint is simply ignored. Why should I need this as I do not change any default configuration here ?

Reply:
ok, I could be wrong, I'm smarter when I run the code myself :wink: